compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUEnums.td

```diff
@@ -131,6 +131,7 @@ class IREEGPU_I32EnumAttr<string name, string summary, list<I32EnumAttrCase> cas
 //     * 0 = CDNA1
 //     * 1 = CDNA2
 //     * 2 = CDNA3
+//     * 3 = CDNA3.5
```

Comment:
I think it's officially called CDNA4?

Reply:
No, I'm pretty sure it's CDNA3.5 - CDNA4's one that isn't in LLVM yet

Reply:
I see it's being called CDNA4 everywhere, e.g., https://www.amd.com/en/newsroom/press-releases/2024-6-2-amd-accelerates-pace-of-data-center-ai-innovation-.html.

Reply:
Ah, so we might have changed names - I saw CDNA3.5 on a lot of internal stuff last year